common/openapi.yml

```diff
@@ -363,9 +363,6 @@ components:
       properties:
         name:
           type: string
-        array_of:
```

Comment:
It looks like this branch is out of date, as this change is already on `main`. Maybe try to rebase your changes on `main` to fix this? Once it's fixed you shouldn't have changes in this file, `DatasetOperations.java`, `RawlsClient.java` and a few more.

Reply:
My guess is this happened because you started work on an existing branch, rather than first checking out `main` and creating a new branch from there. This happens sometimes by accident, I usually fix it by cherry-picking my changes onto a new branch off of `main`.